### DISCUSSION TITLE
3D Visualization

### DISCUSSION THREAD

**Discussion by dzungdinhh** [Project-MONAI/MONAILabel#1471] [Q&A]
Hello, 

If I have a Pytorch model and want to **visualize** the result in 3D, how should I do it in monai label? 

Thank you for your help!

**Comment by diazandr3s** [comment]
Hi @cowfrica-1702,

What type of images are we talking about? Is it CT, MR, Histopathological, or video? 
Do you have access to the labels used to train the PyTorch model? do you think you can retrain the model using a MONAI Label app?

**Reply by dzungdinhh** [reply]
Hi @diazandr3s, thank you for your reply! 

_1. What type of images are we talking about? Is it CT, MR, Histopathological, or video?_
- I am using MRI. 

_2. Do you have access to the labels used to train the PyTorch model?_ 
- Yes, I have access to the labels for the PyTorch. I have full access to the PyTorch model. 

_3. do you think you can retrain the model using a MONAI Label app?_
- No, we mainly aim to use Monai Label for visualization, and we are trying to get it done on 3D Slicer.

**Reply by lassoan** [reply]
To see the segmentation result in 3D, you can click the "Show 3D" button in Segment Editor.

**Reply by dzungdinhh** [reply]
@lassoan, thank you, the Show 3d did work for me. 

But how can I view the slice of each segmentation in 2D? When I drag the 3d input into the subwindow for 2d view, it doesn't work for me.

**Reply by lassoan** [reply]
By default the segmentation is shown in 2D as well. You can hold down the Shift key while moving the mouse in the 3D view (no need to click) to jump the slice views to the moues pointer position in slice views.

**Reply by dzungdinhh** [reply]
It doesn't work for me, as I cannot see the 2D. FYI, my segmentation is in nrrd format, with the shape (512, 512, 4).
